### PR TITLE
Feature/rm 9102 remove scrollbar on resolve border element a

### PR DESCRIPTION
**Pull request by KuJo7**
https://re-motion.atlassian.net/browse/RM-9102

### CODE REVIEW

**Review by MichaelKetting — CHANGES_REQUESTED** [review]
@KuJo7 from reading the code, it loks like BorderELementA and BorderElementB have been completely removed in the test code. If so, the associated images are obsolete and should be deleted. Please verify and if so, remove.

cc @Frazerus in case I missed something.

**Comment by Frazerus** [comment]
These are currently 2 PRs in one, the ResolveBorderElementB is here: https://github.com/re-motion/Framework/pull/1032

**Review by MichaelKetting — APPROVED** [review]

File: Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.Shared.cs [hunk 1]
```diff
@@ -296,13 +296,40 @@ public void ResolveAbsolutePositionedElement ()
...
+            var element = home.Scope.FindId($"borderElement{testSide}").ForElementScopeScreenshot();
+
+            Helper.RunScreenshotTestExact<IFluentScreenshotElement<ElementScope>, ScreenshotTest>(element, ScreenshotTestingType.Both, test, memberName: testName);
+          },
+          2);
+    }
+
+    [Category("Screenshot")]
+    [Test]
+    public void ResolveBorderElementLeft ()
```
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.Shared.cs:326]: Should be done like ResolveborderElement in line 301, or, possibly, merged with the other test.
@Frazerus

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Comment by KuJo7** [comment]
I found some Screenshots from the other task which also should be deleted. I did this within this task, hope that's fine!

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**